src/main/kotlin/com/team4099/robot2023/subsystems/drivetrain/gyro/GyroIOPigeon2.kt

```diff
@@ -18,9 +17,8 @@ import kotlin.math.IEEErem
 
 object GyroIOPigeon2 : GyroIO {
   private var pigeon2 = Pigeon2(Constants.Gyro.PIGEON_2_ID, Constants.Universal.CANIVORE_NAME)
-  private val xyzDps = DoubleArray(3)
 
-  private val isConnected = pigeon2.lastError.equals(ErrorCode.OK)
```

Comment:
I couldn't find an equivalent to this in pigeon2 code, im not sure if the .upTime attribute is the write way to check if the pigeon is connected